### DISCUSSION TITLE
Rename default grouping folders?

### DISCUSSION THREAD

**Discussion by Manfred282** [aldostools/webMAN-MOD#576] [Q&A] [answered]
Hi there,

is it possible toe rename the default grouping folders (e.g. PLAYSTATION®3, PLAYSTATION®2, etc) under My webMAN Games?

I tried to edit the mygames.xml, there are the a entry about that, but everytime i change it, it reverts back to standard after a reboot. I checked "disable content scan on startup", but all changes i made to the mygames.xml are not saved at all.

Something im doing wrong or is it not possible the rename these folders?

many thx in advance ...

**Comment by aldostools** [comment]
The name of the main folder "webMAN Games" can be renamed in the language files (look for the entry "STR_MYGAMES").
However these files are replaced every time the plugin is updated. Only the "Custom" language LANG_XX.TXT is not replaced.

The sub-folder names "PLAYSTATION©3", "PLAYSTATION©2", etc. are hardcoded and cannot be changed, unless you recompile the plugin or hex edit the ELF and resign it back to sprx.

If you edited the mygames.xml, checked the option "disable content scan on startup" and saved the settings, the changes should stay until you refresh the XML again.

**Reply by Manfred282** [reply]
Hi aldo,

just 2 more questions, the language files to edit are the ones under "dev_hdd0/tmp/wm_lang"? Thats the ony language files i found on my hdd, the temp folder just irritates me ;)

2nd question, under settings -> webMAN MOD settings i do select "deactivate multiMAN covers : ICON0.PNG ... Problem, the ICON0.PNG images are squeezed to a square format, like 128x128 pix, that looks really awkward. Is there a way to fix it by myself so that they are shown in its regular rectangular format with 320x176 pix?

thx again ...

**Comment by aldostools** [comment, marked as answer]
There are language files in:
/dev_hdd0/tmp/wm_lang  <- LANG_**.TXT has most string texts used by the plugin
/dev_hdd0/game/XMBMANPLS/USRDIR/FEATURES/webMAN_**.xml <- webMAN Setup menu on XMB

Some texts are hardcoded in the the plugin.

The icons used by XMB web links are squares only. It's recommended that install the covers packs.
The ICON0.PNG is used mainly as fallback when the covers are not found.
